sdk/storage/Azure.Storage.Common/README.md

```diff
@@ -10,55 +10,70 @@ and Azure Queues.
 The Azure.Storage.Common library provides infrastructure shared by the other
 Azure Storage client libraries.
 
-[Source code][source] | [Package (NuGet)][package] | [API reference documentation][rest_docs] | [Product documentation][product_docs]
+[Source code][source] | [Package (NuGet)][package] | [API reference documentation][docs] | [REST API documentation][rest_docs] | [Product documentation][product_docs]

```

Comment:
Is the idea for the API reference docs to go to generic Azure.Storage instead of Common?